src/store/modules/DocumentsRequiredModule.js

```diff
@@ -155,6 +157,15 @@ class DocumentsRequiredModule extends VuexModule {
     );
   }
 
+  /**
+   * @returns {Promise<void>}
+   */
+  @Action
+  async reloadDocumentsStatuses() {
+    await this.loadDocuments();
+    this.updateDocTypeToStatus();
```

Comment:
should we use getter?

Reply:
@lamartire updated to getters and added tests for some cases
ready for review again